### PR TITLE
[HMC-13] - Slicing Home Landing Page

### PR DESCRIPTION
**Pull request by FarhanSaleh**
## Description :

- Add Hero Section
- Add Social Media Constant
- Add Social Media Type 

## Screenshots / Video Record:

Screenshot (1374)
Screenshot (1375)

### CODE REVIEW

File: src/features/Main/sections/Hero/index.tsx [hunk 1]
```diff
@@ -1,6 +1,38 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import HeroImage from '../../../../../public/assets/img/Hero.svg';
```
Comment (vickyadrii) [inline src/features/Main/sections/Hero/index.tsx:3]: saya ada buat config untuk alias path supaya tidak panjang begini kalau import file. Nanti pull ulang dari development ee

cara pakainya kek begini: 
`import test from '@/assets/img/Hero.svg'`

File: src/features/Main/sections/Hero/index.tsx [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
 const Hero: FC = () => {
-  return <div>Hero</div>;
+  return (
+    <div className="container mx-auto px-4 flex flex-col md:flex-row items-center justify-between my-6">
+      <div className=" md:w-2/5 flex flex-col gap-6">
+        <h1 className="text-5xl font-semibold text-[#0F3665]">
+          Improve Your Skill by Talking High-Quality Classes
+        </h1>
+        <p className="text-sm">
+          <strong>HMCLearn</strong> is a streamlined and user-friendly Learning
```
Comment (vickyadrii) [inline src/features/Main/sections/Hero/index.tsx:16]: mungkin prefer pakai tag `<b>` :)

reference: https://www.seobility.net/en/wiki/Strong_and_Bold_Tags#:~:text=The%20difference%20between%20the%20two,word%20or%20section%20of%20text.

Comment (FarhanSaleh) [reply src/features/Main/sections/Hero/index.tsx:16]: okey kak

**Review by vickyadrii — APPROVED** [review]

**Review by akwanmaroso — APPROVED** [review]
good

**Comment by mpsalunggg** [comment]
@FarhanSaleh I think the header of text should be minimized a little more when responsive

File: src/features/Main/sections/Hero/index.tsx [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+import HeroImage from '../../../../../public/assets/img/Hero.svg';
 import { FC } from 'react';
 
+
 const Hero: FC = () => {
-  return <div>Hero</div>;
+  return (
+    <div className="container mx-auto px-4 flex flex-col md:flex-row items-center justify-between my-6">
+      <div className=" md:w-2/5 flex flex-col gap-6">
+        <h1 className="text-5xl font-semibold text-[#0F3665]">
```
Comment (mpsalunggg) [inline src/features/Main/sections/Hero/index.tsx:11]: @FarhanSaleh I think the header of text should be minimized a little more when responsive

Comment (FarhanSaleh) [reply src/features/Main/sections/Hero/index.tsx:11]: ok i'll try it

Comment (mpsalunggg) [reply src/features/Main/sections/Hero/index.tsx]: @FarhanSaleh next when update the ui, u should update the description on the pull request again, lets merge

**Review by mpsalunggg — APPROVED** [review]

**Pull request merged**